modules/ROOT/pages/configure-horizontal-autoscaling.adoc

```diff
@@ -0,0 +1,205 @@
+= Configure Horizontal Autoscaling (HPA) for Runtime Fabric Deployments
+
+Runtime Fabric instances support horizontal scaling of Mule app deployments by spinning up additional replicas or spinning down unneeded replicas in response to the CPU usage as a signal.
+
+Mulesoft supports configuring Mule apps to be responsive to the CPU usage on Kubernetes (K8s) runtime side. You can enable CPU based horizontal autoscaling per application on the Runtime Manager application deployment page, which will result in automatic scale up or scale down of the deployment replicas as needed.
+
+In Kubernetes, a Horizontal Pod Autoscaler (HPA) automatically updates a workload resource, with the aim of automatically scaling the workload to match demand. Horizontal scaling means that the response to increased load is to deploy more pods.
+
+[NOTE]
+HPA is supported only on Runtime Fabric agent version 2.5.0.
+
+
+== Before You Begin
+
+To configure autoscaling, the cluster administrator must enable the https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#support-for-metrics-apis[support for metrics APIs^] on your managed K8s API server.
+Refer to your managed vendors documentation on how to install or enable metrics API servers.
+
+
+== Configuration Recommendations for Cluster Autoscaling
+
+The following recommendations are for your infrastructure team to configure cluster autoscaling
+
+=== Configure Node Groups
+
+Configure one node group per availability zone, with a minimum of one node in the node group.
+This ensures that Kubernetes has information about all the availability zones in the cluster and avoids the issue where zone topology unschedulable pod fails to scale-out topologySpread predicate check.
+
+Make sure that nodes from all the node groups have the exact same labels except for the common labels and specific cloud provider labels that are different across nodes or availability zones by default.
+
+Custom labels that are not in the following list and different across nodes need to be excluded from cluster autoscaler when evaluating similar node groups.
```

Comment:
This is too specific statement, lets make it generic like
```
Custom labels (which are not common labels and specific cloud provider labels) and are different across nodes need to be excluded from cluster autoscaler when evaluating similar node groups.

For reference on managed EKS the labels such as:
...

```

@MichelleZMY just for 2nd opinion